homeassistant/components/tomorrowio/sensor.py

```diff
@@ -128,7 +123,6 @@ def convert_ppb_to_ugm3(molecular_weight: int | float) -> Callable[[float], floa
     TomorrowioSensorEntityDescription(
         key="pressure_surface_level",
         attribute=TMRW_ATTR_PRESSURE_SURFACE_LEVEL,
-        name="Pressure (Surface Level)",
```

Comment:
Used a device class translation here. Please let me know if there is a significant change with pressure and pressure (surface level)/